Formula/a/astro.rb

```diff
@@ -32,8 +33,8 @@ def install
     version_output = shell_output("#{bin}/astro version")
     assert_match("Astro CLI Version: #{version}", version_output)
 
-    run_output = shell_output("echo 'y' | #{bin}/astro dev init")
-    assert_match(/^Initializing Astro project*/, run_output)
+    run_output = shell_output("mkdir /tmp/astro-project && cd /tmp/astro-project && #{bin}/astro dev init")
```

Comment:
would this be good enough?

```suggestion
    run_output = shell_output("#{bin}/astro dev init")
```

Reply:
Astro CLI docs for reference around the actual usage: https://www.astronomer.io/docs/astro/cli/develop-project#create-an-astro-project

Reply:
can it be configured to use `testpath` rather than `tmp` then?

Reply:
good callout, have updated the test to use `testpath`